hub/core/chunk_engine/util.py

```diff
@@ -0,0 +1,75 @@
+import os
+import numpy as np
+from hub import constants
+
+from typing import Tuple
+
+
+def array_to_bytes(array: np.ndarray) -> bytes:
+    return array.tobytes()
+
+
+def get_random_array(shape: Tuple[int], dtype: str) -> np.ndarray:
```

Comment:
Is this used for anything other than testing?

Reply:
true